tests/test_build_latex.py

```diff
@@ -859,31 +860,30 @@ def test_latex_table(app, status, warning):
     # table having :widths: option
     table = tables['table having :widths: option']
     assert ('\\noindent\\begin{tabular}{|\\X{30}{100}|\\X{70}{100}|}' in table)
-    assert ('\\hline\n'
-            '\\sphinxstylethead{\\relax \nheader1\n\\unskip}\\relax &'
-            '\\sphinxstylethead{\\relax \nheader2\n\\unskip}\\relax' in table)
-    assert ('\\hline\ncell1-1\n&\ncell1-2\n\\\\' in table)
-    assert ('\\hline\ncell2-1\n&\ncell2-2\n\\\\' in table)
-    assert ('\\hline\ncell3-1\n&\ncell3-2\n\\\\' in table)
     assert ('\\hline\n\\end{tabular}' in table)
 
+    # table having :align: option (tabulary)
+    table = tables['table having :align: option (tabulary)']
+    assert ('\\begin{table}[htb]\\centering\n'
+            '\\noindent\\begin{tabulary}{\\linewidth}{|L|L|}\n' in table)
```

Comment:
There might be no effect in this case... The table is too wide to align holizontal.